uber/site_sections/preregistration.py

```diff
@@ -236,19 +249,16 @@ def prereg_payment(self, session, payment_id, stripeToken):
         raise HTTPRedirect('paid_preregistrations?payment_received={}', charge.dollar_amount)
 
     def paid_preregistrations(self, session, payment_received=None):
-        if not self.paid_preregs:
-            raise HTTPRedirect('index')
-        else:
-            preregs = [session.merge(Charge.from_sessionized(d)) for d in self.paid_preregs]
-            for prereg in preregs:
-                try:
-                    session.refresh(prereg)
-                except:
-                    pass  # this badge must have subsequently been transferred or deleted
-            return {
-                'preregs': preregs,
-                'total_cost': payment_received
-            }
+        preregs = [session.merge(Charge.from_sessionized(d)) for d in self.paid_preregs]
+        for prereg in preregs:
+            try:
+                session.refresh(prereg)
+            except:
+                pass  # this badge must have subsequently been transferred or deleted
+        return {
+            'preregs': preregs,
+            'total_cost': payment_received
+        }
```

Comment:
Looks like this is a companion change to the one just above, where it redirects from the form straight to the prereg page. This change removes the `if not self.paid_preregs` check (hard to tell cause of the diff), a check which naturally would prevent you from directing someone right from the form to the paid_prereg paid without putting anyone in `self.paid_preregs`.

Hopefully that makes sense. In any case, I think we should add this check back when we revert the change above at the end of `def form`.